cnd/CHANGELOG.md

```diff
@@ -9,194 +9,180 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- **Breaking (Config file format):** Expand `http_api` section in config file to contain both the socket and CORS settings.
 
 ### Added
 
--   Return Ethereum `chain_id` on the HTTP API.
--   Support Ethereum `chain_id` in the Swap Request (HTTP API).
--   Ability to set CORS allowed origins through the configuration file.
--   Added command line option `--dump-config` to print the running configuration to stdout.
+- Return Ethereum `chain_id` on the HTTP API.
+- Support Ethereum `chain_id` in the Swap Request (HTTP API).
+- Ability to set CORS allowed origins through the configuration file.
+- Added command line option `--dump-config` to print the running configuration to stdout.
 
 ### Fixed
 
--   Error responses now properly identify themselves as `application/problem+json`. They have been conforming to this format for a while already, we just never set the `Content-Type` header properly. From now on, applications can fully rely on the error format!
+- Error responses now properly identify themselves as `application/problem+json`. They have been conforming to this format for a while already, we just never set the `Content-Type` header properly. From now on, applications can fully rely on the error format!
 
 ## [0.3.0] - 2019-10-02
 
 ### Changed
 
--   Embed btsieve as a library inside cnd: From now on, you'll only need to run cnd to use COMIT.
+- Embed btsieve as a library inside cnd: From now on, you'll only need to run cnd to use COMIT.
 
 ## [0.2.1] - 2019-09-24
 
 ### Changed
 
--   Use the same Swap ID to identify a swap for both parties.
+- Use the same Swap ID to identify a swap for both parties.
 
 ## [0.2.0] - 2019-09-13
 
 ### Changed
 
--   Statically link openssl in the release build to allow the binaries to be ran out-of-the-box on most Linux distros.
--   Replace ZMQ by using bitcoind's HTTP API for retrieving bitcoin blocks.
+- Statically link openssl in the release build to allow the binaries to be ran out-of-the-box on most Linux distros.
+- Replace ZMQ by using bitcoind's HTTP API for retrieving bitcoin blocks.
 
 ## [0.1.0] - 2019-09-05
 
 ### Added
 
--   All the code since the dawn of comit-rs.
--   Check if btsieve's version matches the expected version, on every request.
--   Ping btsieve on cnd startup, checking for presence and version compatibility.
+- All the code since the dawn of comit-rs.
+- Check if btsieve's version matches the expected version, on every request.
+- Ping btsieve on cnd startup, checking for presence and version compatibility.
 
 ### Changed
 
--   Move config files to standard location based on platform (OSX, Windows, Linux).
--   Align implementation with RFC-002 to use the decision header instead of status codes.
+- Move config files to standard location based on platform (OSX, Windows, Linux).
+- Align implementation with RFC-002 to use the decision header instead of status codes.
 
 [Unreleased]: https://github.com/comit-network/comit-rs/compare/cnd-0.9.0...HEAD
```

Comment:
Same issue than for the list, I believe the release github action added the new lines.